Presentation.Web/app/components/local-config/import/fk-organization-import-change-log.view.html

```diff
@@ -0,0 +1,18 @@
+﻿<table class="table table-striped">
+    <thead>
+        <tr>
+            <th width="33%">Ansvarlig</th>
+            <th width="33%">Number of updates</th>
+            <th width="33%">Update date</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>{{ ctrl.responsibleEntityText }}</td>
+            <td>{{ ctrl.changeLog.consequences.length }}</td>
+            <td>{{ ctrl.logTime }}</td>
+        </tr>
+</table>
+<div class="margin-top-sm">
+    <fk-orgnization-import-consequences-log consequences="ctrl.changeLog.consequences"></fk-orgnization-import-consequences-log>
```

Comment:
no need to have both tags.. just add a closed tag:
`<fk-orgnization-import-consequences-log consequences="ctrl.changeLog.consequences" />`

Reply:
also use data- prefix to the consequences attr